pkg/detectors/netlify/v2/netlify_v2.go

```diff
@@ -21,9 +22,10 @@ var _ detectors.Detector = (*Scanner)(nil)
 var _ detectors.Versioner = (*Scanner)(nil)
 
 var (
-	client = common.SaneHttpClient()
-
-	keyPat = regexp.MustCompile(detectors.PrefixRegex([]string{"netlify"}) + `\b(nfp_[a-zA-Z0-9_]{36})\b`)
+	client           = common.SaneHttpClient()
+	rotationGuideUrl = "https://howtorotate.com/docs/tutorials/netlify/"
```

Comment:
suggestion: I like the idea of extracting the string literals. I think we could use `const` for these two new variables, since they don't need to change. Similar comment for the v1 detector.